src/store/index.js

```diff
@@ -41,10 +41,6 @@ const state = {
    * Application base URL (set by the backend).
    */
   baseUrl: '/',
-  /**
-   * Application environmnet (e.g. offline, development, production), retrieved from NODE_ENV.
-   */
-  environment: process.env.NODE_ENV.toUpperCase(),
```

Comment:
Less one property. We can access the version directly as ES6 imports allow you to import `package.json` directly.